.github/workflows/create-version.yml

```diff
@@ -0,0 +1,53 @@
+name: Create New Version
+run-name: Creating new ${{ github.event.inputs.version }} ${{ github.event.inputs.channel }} version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+        default: minor
+      channel:
+          description: Channel
+          type: choice
+          options:
+            - release
+            - beta
+          default: release
+
+env:
+  CI: true
+
+permissions:
+  contents: write
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.BCC_BOT_ACCESS_TOKEN }}
```

Comment:
Be sure to have this token. IIRC it is a bit of a pain to create one as it is per repository (can't reuse an existing one)